### ISSUE TITLE
Let's do Meadow pine tree support

### ISSUE THREAD

**Issue by benio1394** [hammertater/treechop#276]
Hello, I want to report that pine tree from let's do meadow mod is not supported by tree chop. I ask about this satisfy, but he ask in return to rather report this here.

**Comment by hammertater** [comment]
Sorry for the delay! In Minecraft 1.19.2, the problem is that Meadow's leaves blocks don't have the `#minecraft:leaves` tag. I see that it's fixed in Minecraft 1.20.1.

You can fix the issue yourself in 1.19.2 in the treechop-common.toml config file by telling TreeChop about Meadow's leaves blocks. Find these lines:


```toml
	[treeDetection.leaves]
		#Blocks that should be considered leaves
		#Specify using registry names (mod:block), tags (#mod:tag), and namespaces (@mod)
		blocks = ["#treechop:leaves_like", "#minecraft:leaves", "#minecraft:wart_blocks", "#forge:mushroom_caps", "minecraft:shroomlight"]
```

and add "meadow:pine_leaves" and "meadow:alpine_birch_leaves_hanging" to the blocks list, like this:

```toml
		blocks = ["meadow:pine_leaves", "meadow:alpine_birch_leaves_hanging", "#treechop:leaves_like", "#minecraft:leaves", "#minecraft:wart_blocks", "#forge:mushroom_caps", "minecraft:shroomlight"]
```

**Comment by hammertater** [comment]
Related - https://github.com/satisfyu/Let-s-Do-Hub/issues/8

**Comment by hammertater** [comment]
Closing, since this is fixed as of Meadow 1.2.9
